server_http/endpoints.py

```diff
@@ -0,0 +1,18 @@
+import markupsafe
+from flask import Flask, render_template
+
+from src import enums
+
+app = Flask(__name__)
+
+
+@app.route('/', methods=['GET'])
+@app.route('/<chat>', methods=['GET'])
+def open_chatbox(chat=enums.Constants.DEFAULT_CHAT_NAME):
+    requested_chat = markupsafe.escape(chat)
+    print('chat:', requested_chat)
```

Comment:
Debug print should be removed (or replaced with the existing logger if the message can be useful).